pkgs/development/tools/rye/default.nix

```diff
@@ -10,24 +10,27 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rye";
-  version = "0.6.0";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "mitsuhiko";
     repo = pname;
-    rev = version;
+    rev = "refs/tags/${version}";
     # One of the tests runs git command so we need .git directory there
     leaveDotGit = true;
```

Comment:
Could we just skip the git-dependent test?

Reply:
It seems that all the tests now work. I removed the `leaveDotGit = true;` line and did not have to skip any test !

Reply:
> It seems that all the tests now work. I removed the leaveDotGit = true; line and did not have to skip any test !

Well no. While updating to 0.11.0, I was needed to skip the test.